### ISSUE TITLE
Cannot import files from SC:BW installation

### ISSUE THREAD

**Issue by marmistrz** [Wargus/stargus#21]
I'm using a local installation of SC:BW. Startool cannot import the data due to case mismatch:

```
$ startool /run/media/marcin/eHDD/wine/StarCraft 
Extract from "/run/media/marcin/eHDD/wine/StarCraft" to "data"
Using mpq list file "data/mpqlist.txt"
Please be patient, the data may take a couple of minutes to extract...
Archive "/run/media/marcin/eHDD/wine/StarCraft/install.exe"
Can't open /run/media/marcin/eHDD/wine/StarCraft/install.exe
Could not open archive "/run/media/marcin/eHDD/wine/StarCraft/install.exe", skipping
Archive "/run/media/marcin/eHDD/wine/StarCraft/starcraft.mpq"
Can't open /run/media/marcin/eHDD/wine/StarCraft/starcraft.mpq
Could not open archive "/run/media/marcin/eHDD/wine/StarCraft/starcraft.mpq", skipping
Archive "/run/media/marcin/eHDD/wine/StarCraft/stardat.mpq"
Can't open /run/media/marcin/eHDD/wine/StarCraft/stardat.mpq
Could not open archive "/run/media/marcin/eHDD/wine/StarCraft/stardat.mpq", skipping
Fatal error: Cannot extract data
$ ls /run/media/marcin/eHDD/wine/StarCraft
battle.snp    BroodWar.mpq   License.html  Patch_rt.mpq              save          SEditFRA.loc  standard.snp                  Starcraft.mpq  StarEdit.hlp
bnupdate.exe  characters     License.txt   patch.txt                 SEditDEU.loc  SEditITA.loc  StarCraft.exe                 StarDat.mpq    storm.dll
bnupdate.log  EditLocal.dll  Local.dll     'Register Starcraft.url'  SEditENU.loc  SEditPTB.loc  'StarCraft Install Log.html'  StarEdit.cnt
BrooDat.mpq   icon           Maps          Riched20.dll              SEditESP.loc  Smackw32.dll  'StarCraft Manual.pdf'        StarEdit.exe
```

**Comment by ElectricPrism** [comment]
Correct me if I'm wrong but Stargus isn't designed to import from the BroodWar game?

Also I believe you're supposed to import from /path/to/starcraft/cd/ andn ot a wine installation.

I was able to get Stargus working a couple months ago so I'm trying to iron out the Linux stuff myself on Arch.

Here's what I got so far
https://ptpb.pw/SX9F

I think the LUA scripts might need some modding to get it running (this was the case about a year ago)
